CsvParser/csvParser.go

```diff
@@ -85,4 +86,45 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
 	return incidentArray
 }
 
- 
\ No newline at end of file
+func convertDefaultIncidentToIncident(arr []incident.DefaultIncident) []incident.Incident {
+	incidentArr := make([]incident.Incident, len(arr), len(arr))
+	for i := range arr {
+		incidentArr[i] = &arr[i]
+	}
+	return incidentArr
+}
+
+func makeJsonObjFile(arr []incident.Incident) *os.File {
+	// numObjects determines how many incidents are stored in the json
+	const numObjects = 1
+	f, err := os.Create("incidents.json")
+	var objs [numObjects]incident.IncidentData
+	if err != nil {
+		fmt.Println(err)
+		return f
+	}
+
+	for i := 0; i < numObjects; i++ {
+		gpStart, gpEnd := arr[i].GetGoodPeriod()
+		bpStart, bpEnd := arr[i].GetBadPeriod()
+		s := arr[i].GetSeverity()
+		ta := arr[i].GetTestsAffected()
+		gpi := arr[i].GetGoodPeriodInfo()
+		bpi := arr[i].GetBadPeriodInfo()
+		ii := arr[i].GetIncidentInfo()
+		inc := incident.IncidentData{gpStart, gpEnd, bpStart, bpEnd, s, ta, gpi, bpi, ii}
+		objs[i] = inc
+	}
+	b, err := json.Marshal(objs)
```

Comment:
try coming up with variable names that are longer than one or two characters

Reply:
Changed var names (except for n because it is only for file write errors anyways)